Samples/JavaKitSampleApp/Sources/JavaKitExample/Java2Swift.config

```diff
@@ -0,0 +1,5 @@
+{
+  "classes" : {
+    "java.util.Vector" : null
```

Comment:
Vector is a rarely used type that folks generally avoid as it predates the "proper collections framework", I'd recommend avoiding it in examples (it'll look weird to Java developers that we're using that anywhere) and using `java.util.ArrayList` in examples

Reply:
I was picking something silly that we wouldn't actually end up wrapping in any of the JavaKit* modules, but yeah... Vector is extra silly. Thanks for the ArrayList suggestion.

Reply:
Yeah, no harm but a bit too silly maybe :) No prob